projects/cashmere/src/lib/form-field/hc-form-field.component.html

```diff
@@ -0,0 +1,32 @@
+<div [ngClass]="{ 'hc-form-field-wrapper': !inline, 'hc-form-field-wrapper-inline': inline }">
+    <span [ngClass]="{ 'hc-form-field-label-wrapper': !inline, 'hc-form-field-label-wrapper-inline': inline }">
```

Comment:
just a coding style preference on my part, but this might be cleaner:
`[ngClass]="inline ? 'hc-form-field-label-wrapper-inline' : 'hc-form-field-label-wrapper'"`

Reply:
ah, yes that's way better - good call